docs/md/modules-covid.md

```diff
@@ -0,0 +1,166 @@
+# Covid namespace
+
+WorkFlow Launcher (WFL) runs within the `covid` namespace to process workflows using the `Sarscov2IlluminaFull` pipeline. 
+
+Workflow Launcher uses a model which includes a source, a sink and an executor. Within the `covid` namespace, the source at the time of this writing is either directly from the Terra Data Repository (`TerraDataRepo`), or a list of snapshots which are passed to WFL (`TDR Snapshots`). 
+
+### API
+
+The `covid` module supports the following API endpoints:
+
+| Verb | Endpoint                            | Description                                                              |
+|------|-------------------------------------|--------------------------------------------------------------------------|
+| GET  | `/api/v1/workload`                  | List all workloads, optionally filtering by uuid or project              |
+| POST | `/api/v1/create`                    | Create a new workload                                                    |
+| POST | `/api/v1/start`                     | Start a workload                                                         |
+| POST | `/api/v1/stop`                      | Stop a running workload                                                  |
+| POST | `/api/v1/exec`                      | Create and start (execute) a workload                                    |
+
+Although it is possible to be run differently, the typical life-cycle of a workload is a multi-stage process:
+
+1. The caller needs to create a workload and specify the source, sink and executor. The source must be either a snapshot or a list of snapshots. If the former, then the type is `TDR Snapshots`. If the latter, than it is of type `TDRSnapshotListSource`. The executor must be of type `TerraExecutor`, and the sink must be of type `TerraWorkspaceSink`. If everything else passes verification, then the caller will receive a response which will contain the `uuid` of the workload.
+   
+2. Next, the caller needs to "start" the newly created workload, which will begin the analysis. 
+   
+3. WFL can, in addition, stop watching a workflow using the `stop` endpoint. This will not cancel analysis, but WFL will stop managing the workload.
```

Comment:
Worth elaborating slightly on what "WFL will stop managing the workload" entails.  Something about no longer polling for new inputs?